bin/blockfrostUtils.js

```diff
@@ -0,0 +1,74 @@
+
+import { readFile } from 'fs/promises';
+import { BlockFrostAPI } from "@blockfrost/blockfrost-js";
+
+/**
+ * 
+ * @param {*} blockfrostKey 
+ * @param {*} walletAddress 
+ * @returns 
+ */
+const queryUTxOviaBlockfrost = async (blockfrostKey, walletAddress) => {
+  const blockfrost = new BlockFrostAPI({
+    projectId: blockfrostKey,
+  });
+  const utxos = await blockfrost.addressesUtxosAll(walletAddress);
+  let TxHash = "";
```

Comment:
use first letter always lower case for variables 🙏🏻 

Check https://www.w3schools.com/js/js_conventions.asp